### PR TITLE
feat(spdk): updating to SPDK 23.05

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

**Comment by dsavitskiy** [comment]
bors try

**Comment by dsavitskiy** [comment]
bors try

**Comment by dsavitskiy** [comment]
bors try

File: ci.nix [hunk 1]
```diff
@@ -70,6 +70,10 @@ mkShell {
   ETCD_BIN = "${etcd}/bin/etcd";
 
   shellHook = ''
+    ${pkgs.lib.optionalString (!nospdk) "echo 'SPDK version    :' $(echo $SPDK_PATH | sed 's/.*libspdk-//g')"}
```
Comment (tiagolobocastro) [inline ci.nix:74]: Should this be added to shell.nix as well?

**Review by tiagolobocastro — APPROVED** [review]

**Review by hrudaya21 — APPROVED** [review]

File: ci.nix (hunk 1 above)
Comment (dsavitskiy) [reply ci.nix:74]: done

**Comment by dsavitskiy** [comment]
bors merge

**Pull request merged**